UsageDoc/CA_UsageDocument.md

```diff
@@ -292,6 +330,11 @@ argument list.
     
     **Example** : `docker run --rm -it -v /D/Projects/Output:/mnt/Output -v /D/Projects/DockerLog:/var/log -v /D/Projects/CAConfig:/etc/CATool clearingautomationtool dotnet ArtifactoryUploader.dll --settingsfilepath /etc/CATool/appSettings.json --mode test`
 
+..or,
+
+   **Example** : `ArtifactoryUploader.exe --settingsfilepath /<PathToConfig>/appSettings.json --mode test`
+
+
```

Comment:
Add the issue you get while running with md servers in the trouble shoot section along with the solution